<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SizeCapture</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(34,40,49)</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>800</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>700</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif CJK KR Black</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(57, 62, 70);
border-radius: 20px;
padding: 5px;
</string>
    </property>
    <widget class="QLabel" name="Image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>700</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>700</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>700</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(57, 62, 70);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Measure">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>630</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK HK Black</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color:rgb(0, 123, 131);
}
QPushButton{
	background-color:rgb(0, 173, 181);
	border-radius: 10px;
	padding: 5px;
}
</string>
     </property>
     <property name="text">
      <string>측정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="file_search">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover{
	background-color:rgb(0, 123, 131);
}
QPushButton{
	background-color:rgb(0, 173, 181);
	border-radius: 10px;
	padding: 5px;
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>serach.png</normaloff>
       <normalon>serach.png</normalon>serach.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QFrame" name="toggle_frame">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:#FFFFFF;
	border-radius: 15px;
	padding: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="toggle">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>26</width>
        <height>26</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: rgb(0, 123, 131);
    border-radius: 13px;
    border-style: outset;
    background: rgb(0, 123, 131);
    padding: 5px;
    }


</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="heightInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#FFFFFF;
border-radius: 10px;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>180</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
